---
layout: archive
title: "Research"
permalink: /research/
author_profile: true
---

<ins> Methodology </ins>  

<span style="font-size:0.8em;">
Trained as a statistician, I have long standing interest in statistical methodology development for graphical models, either directed graphs, e.g. directed acyclic graphs (DAG) or undirected graphs, e.g. Gaussian graphical models  as they serve a compact and elegant way to decode and visualize complex relations among random variables and thus generate wide applications in many fields, such as social network, biological pathway analysis and Gaussian Copula graphical model in financial market. Most applications center around graphical models. I am also generally interested in developing novel statistical methods(frequentist approach or Bayesian method) driven by problems arising in biostatistics, public health, (epi)genetics and genomics.</span>    

<ins>Collaboration </ins>

<span style="font-size:0.8em;">
  As a Biostatistician in our dental school, I am excited to work with faculties, residents and students to support their projects, thesis, etc by providing statistical analysis and consulting. I am also collaborating with researchers on Marquette campus and beyond in data science, public health and statistical genetics.         
  

## Graphical models and network analysis 


<p align="center">
  <img src="https://han16.github.io/shengtonghan.github.io/images/network.png?raw=true" alt="Photo" style="width: 450px;"/> 
</p>


<span style="font-size:0.8em;">
Graphical models have attracted increasing attentions in computer science, machine learning and artificial intelligence. One challenging problem is developing computationally efficient algorithms with nice statistical property to infer graph structure which allows one to identity conditional (in)dependence relations among variables. The computing burden is even more severe for directed graphs as directionality brings complexity in. We are developing novel statistically robust and computationally efficient machine learning algorithms to address these challenges.        </span> 


<span style="font-size:0.9em;"> 
<b>  Gaussian Bayesian network comparisons with graph ordering unknown </b> <br></span>
<span style="font-size:0.7em;">
Hongmei Zhang, Xianzheng Huang, <b>Shengtong Han</b>, Faisal I. Rezwan, Wilfried Karmaus, Hasan Arshad, John W. Holloway. <i> Computational Statistics and Data Analysis</i>, 2021, 157, 107156. [[PDF]](https://han16.github.io/shengtonghan.github.io/files/Bayesiannetworkcomparison.pdf) [[Link]](https://www.sciencedirect.com/science/article/pii/S0167947320302474) </span>

<span style="font-size:0.9em;"> 
<b>   An efficient Bayesian approach for Gaussian Bayesian network structure learning </b> <br></span> 
<span style="font-size:0.7em;">
<b>Shengtong Han</b>, Hongmei Zhang, Wilfried Karmaus, Graham Roberts, Hasan Arshad. <i>Communications in Statistics-Simulation and Computation</i>, 2017, 46, 5070-5084. [[PDF]](https://han16.github.io/shengtonghan.github.io/files/network2017.pdf)[[Link]](https://www.tandfonline.com/doi/full/10.1080/03610918.2016.1143103?scroll=top&needAccess=true) </span>

<span style="font-size:0.9em;"> 
<b>     A Full Bayesian Approach for Boolean Genetic Network Inference </b> <br></span> 
<span style="font-size:0.7em;">
<b>Shengtong Han</b>, Raymond K. W. Wong, Thomas C. M. Lee, Linghao Shen, Shuo-Yen R. Li, Xiaodan Fan. <i>PLoS ONE</i>, 2014, 9(12): e115806. doi:10.1371/journal.pone. 0115806. [[PDF]](https://han16.github.io/shengtonghan.github.io/files/Booleannetwork2014.pdf)[[Link]](https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0115806&type=printable) </span> 


## Statistical (epi)genetics and genomics 

<p align="center">
  <img src="https://han16.github.io/shengtonghan.github.io/images/genetics.png?raw=true" alt="Photo" style="width: 350px;"/> 
</p>

<span style="font-size:0.8em;">
Rare variants (RV) are believed to have large effect size and have the potential to explain missing heritability in human complex traits such as autism.  Rare variant association tests tend to be underpowered due to low allele frequency and simple pooling of variants with heterogeneous effect size, e.g. LoF variants tend to be more deleterious than missense variants. We are developing a  Bayesian framework to boost the statistical power by accounting for heterogeneous effect size in different variant groups defined based on numerous functional annotations, such as Polyphen, CADD score and apply the method to study the genetics of human complex diseases, such as Autism Spectrum Disorders (ASD) and Alzheimer's diseases (AD).              </span> 



<span style="font-size:0.9em;">
<b>[A Bayesian method for rare variant analysis using functional annotations and its application to Autism](http://han16.github.io/shengtonghan.github.io/publications/MIRAGE)  </b> <br> </span>
<span style="font-size:0.7em;">
 <b>Shengtong Han</b>, Nicholas Knoblauch, Gao Wang, Siming Zhao, Yuwen Liu, Yubin Xie, Wenhui Sheng, Hoang T Nguyen, Xin He. 
[[bioRxiv]](https://www.biorxiv.org/content/10.1101/828061v1.full.pdf)[[R-package]](https://xinhe-lab.github.io/mirage/)
</span>
  
<span style="font-size:0.9em;"> 
<b>    Identifying heterogeneous transgenerational DNA Methylation sites via clustering in Beta regression </b> <br></span> 
<span style="font-size:0.7em;">
<b>Shengtong Han</b>, Hongmei Zhang, Gabrielle A. Lockett, Nandini Mukherjee, John W. Holloway, Wilfried J.J. Karmaus. <i>The Annals of Applied Statistics</i>, 2015, 9, 2052-2072. [[PDF]](https://han16.github.io/shengtonghan.github.io/files/transmissionpaper2015.pdf)[[Link]](https://projecteuclid.org/download/pdfview_1/euclid.aoas/1453994191) </span> 



## Computational Biology and Bioinformatics

<p align="center">
  <img src="https://han16.github.io/shengtonghan.github.io/images/bioinformatics.png?raw=true" alt="Photo" style="width: 450px;"/> 
</p>

<span style="font-size:0.9em;"> 
 <b> Jump-seq: genome-wide capture and amplification of 5hmC sites </b> <br></span> 
<span style="font-size:0.7em;">
Lulu Hu\+, Yuwen Liu\+, <b>Shengtong Han\+</b>, Lei Yang+, Xiaolong Cui, Yawei Gao, Qing Dai, Xingyu Lu, Xiaochen Kou, Yanhong Zhao, Wenhui Sheng, Shaorong Gao, Xin He and Chuan He. <i>Journal of American Chemical Society</i>, 2019, 141, 22, 8694-8697. [[PDF]](https://han16.github.io/shengtonghan.github.io/files/jumpseq2019.pdf)[[Link]](https://pubs.acs.org/doi/pdf/10.1021/jacs.9b02512) </span> 
