---
layout: archive
title: "Software"
permalink: /software/
author_profile: true
---




### Deep-MOTIFs: an R package for a Deep learning framework for <ins>M</ins>ulti-<ins>O</ins>mics in<ins>T</ins>egrat<ins>I</ins>on <ins>F</ins>or auti<ins>S</ins>m

<p align="center">
  <a href="https://github.com/wdi169286/Deep-MOTIFs" target="_blank">
    <img src="https://han16.github.io/shengtonghan.github.io/images/deep-motifs.png?raw=true" 
         alt="Deep-MOTIFs workflow" style="width: 450px;">
  </a>
</p>

<span style="background-color: yellow; color: black;">Deep-MOTIFs</span> (being developed) is a deep learning framework to integrate multi-omics data for predicting risk genes not only for autism, but also for other traits and diseases as long as multi-omics data are available. Basically every gene was characterized with a high dimensional vector,  (tabular data), e.g. with the order of >1,000, consisting of continuous, discrete or whatever format, then Deep-MOTIFs learns the unique features associated with the  <span style="color: red;">known</span> risk genes and then predicts  <span style="color: green;">novel</span> risk genes genome-wide. 



### MIRAGE: an R package for <ins>MI</ins>xture model based <ins>R</ins>are variant <ins>A</ins>nalysis on <ins>GE</ins>nes

<p align="center">
  <a href="https://xinhe-lab.github.io/mirage/" target="_blank">
    <img src="https://han16.github.io/shengtonghan.github.io/images/MIRAGEworkflow.png?raw=true" 
         alt="MIRAGE workflow" style="width: 450px;">
  </a>
</p>

<span style="background-color: yellow; color: black;"> MIRAGE </span> is a new Bayesian statistical method for rare variant (RV) association testing that better accounts for heterogeneity of variant effects within a gene using external functional annotations. It partitions variants into disjoint groups and variants with similar effect size are assigned into one group. MIRAGE estimates the effet size of every group and calculates the likelihood of a gene that is a risk gene by Bayes factor. Summary statistics, i.e. total number of variants in cases and controls, rather than individual level variant information  at every single genomic locus are required as input. The samples need to be matched between cases and controls.   [[read more]](https://xinhe-lab.github.io/mirage/index.html). 



### GEARS: an R package for  <ins>G</ins>aussian differ<ins>E</ins>nti<ins>A</ins>l netwo<ins>R</ins>k analysi<ins>S</ins>



<p align="center">
  <a href="https://github.com/han16/GEARS" target="_blank">
    <img src="https://han16.github.io/shengtonghan.github.io/images/gears.png?raw=true" 
         alt="MIRAGE workflow" style="width: 450px;">
  </a>
</p>





GEARS is a Bayesian approach to reconstruct Gaussian Bayesian network and compare two networks (identical or differential) with graph ordering unknown. [[read more]](https://github.com/han16/GEARS)

### BBN: an R package for <ins>B</ins>ayesian  <ins>B</ins>oolean  <ins>N</ins>etwork inference




<p align="center">
  <a href="https://github.com/han16/BBN" target="_blank">
    <img src="https://han16.github.io/shengtonghan.github.io/images/BNdiagram.JPG?raw=true" 
         alt="MIRAGE workflow" style="width: 450px;">
  </a>
</p>



BBN is a Bayesian approach for Boolean network reconstruction. [[read more]](https://github.com/han16/BBN)


